tasks/main-clevis.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Ensure required packages are installed
+  package:
+    name: "{{ __nbde_client_package_list }}"
+    state: present
+
+
+- name: Prepare keyfiles, perform clevis operations and dispose of keyfiles
+  block:
+    - name: Ensure we transfer keyfiles
+      copy:
+        src: "{{ item.keyfile | default('/dev/null') }}"
```

Comment:
What does it mean to use an empty keyfile?  Is that supported by clevis?  Should this be an error instead? e.g. if the user does not define keyfile, by default Ansible will throw an error here.

Reply:
We need to provide a valid key for the LUKS device, via either "pass", a key file or we can even use already existing bindings and not specify either pass or keyfile. When there are keyfiles, I need to copy them to managed host, but if there are none, I can skip this (and the cleanup) tasks. I will try to use your suggestion to skip these when there is no keyfile specified.